datasets.py

```diff
@@ -583,6 +673,7 @@ class YfccPlacesDataset(data.Dataset):
     def __init__(self, transform, metric=0):
         self.transform = transform
         
+        self.supercategories_to_names = DEFAULT_GROUPINGS_TO_NAMES
```

Comment:
for all of these, I believe they also will need to be accompanied with a "self.group_mapping = group_mapping_creator(self.labels_to_names, self.supercategories_to_names)"?